airbyte-integrations/connectors/destination-iceberg-v2/src/main/kotlin/io/airbyte/integrations/destination/iceberg/v2/io/IcebergUtil.kt

```diff
@@ -185,4 +188,15 @@ object IcebergUtil {
         schema.identifierFieldNames().forEach { builder.asc(it) }
         return builder.build()
     }
+
+    private fun getOperation(record: DestinationRecord, schema: Schema): Operation =
+        if (record.data.toJson().get(AIRBYTE_CDC_DELETE_COLUMN) != null) {
```

Comment:
I wonder how costly this `toJson` method call on each record would be. Do you think we should do this upstream when the destination receives an actual json from the platform?

Reply:
@subodh1810 I was concerned about this too, but I don't think that there is any other way at the moment to discover this.  We can comment this out for now/not support deletes at the moment.  Ideally, we would add this information to the protocol so we can test it directly without serializing the payload.

Reply:
I have disabled the delete check for now and added comments about why to the code